src/bundle/Resources/richtext/stylesheets/docbook/xhtml5/edit/core.xsl

```diff
@@ -317,6 +317,12 @@
           <xsl:value-of select="@ezxhtml:class"/>
         </xsl:attribute>
       </xsl:if>
+      <xsl:if test="@numeration = 'arabic'"><xsl:attribute name="style">list-style-type:decimal;</xsl:attribute></xsl:if>
+      <xsl:if test="@numeration = 'arabicleadingzero'"><xsl:attribute name="style">list-style-type:decimal-leading-zero;</xsl:attribute></xsl:if>
+      <xsl:if test="@numeration = 'lowerroman'"><xsl:attribute name="style">list-style-type:lower-roman;</xsl:attribute></xsl:if>
+      <xsl:if test="@numeration = 'upperroman'"><xsl:attribute name="style">list-style-type:upper-roman;</xsl:attribute></xsl:if>
+      <xsl:if test="@numeration = 'loweralpha'"><xsl:attribute name="style">list-style-type:lower-latin;</xsl:attribute></xsl:if>
+      <xsl:if test="@numeration = 'upperalpha'"><xsl:attribute name="style">list-style-type:upper-latin;</xsl:attribute></xsl:if>
```

Comment:
So if I understand the conversation, it's not possible to disable some options here, but why can't they be defined as-is so you can avoid these ifs?

Reply:
@alongosz 
Because the names doesn't match the DocBook specification : https://tdg.docbook.org/tdg/5.0/orderedlist.html
I can change that in the schema if you prefer though ?

Reply:
> @alongosz Because the names doesn't match the DocBook specification : https://tdg.docbook.org/tdg/5.0/orderedlist.html I can change that in the schema if you prefer though ?

If this is strictly specified let's keep it :+1: